sdk/src/main/java/com/mapbox/maps/MapSurface.kt

```diff
@@ -65,6 +65,7 @@ class MapSurface @JvmOverloads constructor(
    */
   fun surfaceChanged(width: Int, height: Int) {
     renderer.surfaceChanged(surface, width, height)
+    onSizeChanged(width, height)
```

Comment:
can we add a test here?

Reply:
we will add tests in general for `MapSurface` in the separate PR - and then this case will be covered.